src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java

```diff
@@ -562,4 +663,60 @@ public String getStatusByKey(String key) {
 		}
 		return "";
 	}
+
+	/**
+	 * Actualize the {@link MedicalInventory}'s ward.
+	 *
+	 * @param inventory the {@link MedicalInventory}
+	 * @return {@link MedicalInventory}. It could be {@code null}.
+	 * @throws OHServiceException
+	 */
+	public MedicalInventory actualizeMedicalWardInventoryRow(MedicalInventory inventory) throws OHServiceException {
```

Comment:
also this method is still missing tests.
@ArnaudFofou, did you check the tests for the OP-1199 ?